### ISSUE TITLE
I had an error 'Model' object has no attribute 'optimizer'

### ISSUE THREAD

**Issue by xuzhang5788** [surmenok/keras_lr_finder#4]
In your example, your model is Sequential(). I tried it and it works perfectly. However, my model was formed as model = Model(input, output), then I received above error message. Any advice? Thanks a lot.

**Comment by xuzhang5788** [comment]
I solved the problem. I need to compile my model first, then do learning rate finder. Thanks
